setup.cfg

```diff
@@ -0,0 +1,27 @@
+[metadata]
+name = openPMD-validator
+version = 1.0.0
+url = https://github.com/openPMD/openPMD-validator
+# author = ...  # TODO
+# author_email = ...  # TODO
+# maintainer = ...  # TODO
+# maintainer_email = ...  # TODO
+license = ISC
+description = Validator and examples for openPMD format
+classifier =
+    # Development Status :: 4 - Beta
+    Environment :: Console
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: ISC License (ISCL)
+    Operating System :: OS Independent
+    Programming Language :: Python :: 2.7
+    Programming Language :: Python :: 3
+    Topic :: Scientific/Engineering :: Physics
+
+[files]
+packages = opmd_validator
```

Comment:
ideally, `openPMD-validator`

Reply:
I wonder if such a thing such as sub-packages like `openPMD.validator`, `openPMD.viewer`, `openPMD.API`, ... exist?

Reply:
I'm not an expert, but I think that packages are mostly dashed, while modules are nearly universally underscored. Please settle on something and I'll be happy to use them.

Reply:
"I wonder if such a thing such as sub-packages like openPMD.validator, openPMD.viewer, openPMD.API, ... exist?" — Submodules shipped in different packages? Yes, of course it's used (and abused). Bear in mind that adopting this across several projects is a large external-facing change.

Reply:
you are right, dashed name is good and underscored module, just let us use the full name `openPMD_` as prefix please. This is the name we want to change the viewer to as well: https://github.com/openPMD/openPMD-viewer/issues/82

Reply:
Oh, OK. Hope it'd all get squashed on merge.